lib/elixir_console_web/live/console_live.ex

```diff
@@ -82,6 +77,10 @@ defmodule ElixirConsoleWeb.ConsoleLive do
     end
   end
 
+  defp add_command_to_history("", history), do: history
+  defp add_command_to_history(command, []), do: [command]
+  defp add_command_to_history(command, history), do: [command | history]
```

Comment:
Is the second clause needed? Isn't it actually covered by the 3rd clause already?

Reply:
You're right. Fixed